include/z64actor.h

```diff
@@ -357,4 +357,95 @@ typedef enum {
     DOORLOCK_NORMAL_SPIRIT
 } DoorLockType;
 
+typedef enum {
+    NAVI_ENEMY_GOHMA = 1,
+    NAVI_ENEMY_ENGOMA_EGG = 2,
+    NAVI_ENEMY_ENGOMA = 3,
```

Comment:
you can remove the ` = intval` when they aren't needed

Reply:
Uh yeah, I had done that already.. but didn't make it to the PR somehow. Just applied those changes.